frontend/src/__tests__/cypress/cypress/e2e/modelServing/ModelMetrics.cy.ts

```diff
@@ -522,9 +533,13 @@ describe('Model Metrics', () => {
     });
 
     // test service error
-    cy.wait('@getTrustyAIError');
+    cy.wait('@getTrustyAIError').then((interception) => {
+      expect(interception.response?.body).to.eql(mock403Error({}));
+    });
```

Comment:
The response here is coming from our mocks. There is no need to assert that our mocks equal a specific value because they are generated within the test file. We only need to assert the request body.
Please remove.